qcfractal/queue/executor_adapter.py

```diff
@@ -83,8 +83,8 @@ def _submit_task(self, task_spec: Dict[str, Any]) -> Tuple[Hashable, Any]:
         return task_spec["id"], task
 
     def count_running_workers(self) -> int:
-        # Have not worked through this yet for Dask
-        raise NotImplementedError
+        # Note: This number may not quite be right if its treating "worker" as a "job" or Dask-Distributed "worker"
```

Comment:
`self.client._count_active_workers() * self.client.worker_processes` in that case?